hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -50,17 +50,28 @@
       tests: ["i2c_error_intr"]
     }
     {
-      name: stress_all_with_reset
-      desc: '''
-            Pull reset at random times, ensure DUT are reset and recovered/ correctly and
-            there is no residual data left in the registers.
+      name: stress_all
+      desc: ''',
+            Support vseq (context) switching with random reset in between.
 
             Stimulus:
-              - Randomly reset DUT
-              - Restart send/receive transaction
+              - Combine above sequences in one test to run sequentially
+                except csr sequence and i2c_rx_oversample_vseq (requires zero_delays)
+              - Randomly add reset between each sequence,
```

Comment:
do we need this comma? and also line 54.

Reply:
@cindychip definitely not, thank